components/api/src/main/java/com/hotels/styx/api/LiveHttpRequest.java

```diff
@@ -1017,7 +1017,7 @@ private void ensureContentLengthIsValid() {
 
         private static boolean isInteger(String contentLength) {
```

Comment:
So live requests + responses do not have the `>= 0` check that full ones have?

Reply:
My mistake. Got distracted when implementing this. :-/

Added.